components/02-elements/button/Button.vue

```diff
@@ -1,34 +1,26 @@
+<template>
+  <component
+    :is="tag ? tag : 'button'"
+    :class="[tag && 'button',
```

Comment:
`button` class is not conditional

Reply:
Hmm, why not? If `tag === 'button'`, we don't need add this class.